test/scheduleRecurringMetricsGeneration.test.ts

```diff
@@ -0,0 +1,78 @@
+import 'mocha'
+import { assert } from 'chai'
+
+import { AddressCount, Metrics } from '../src'
+
+import config from './config'
+import structuredClone from './helpers'
+
+let metrics: Metrics
+
+describe('Push Metrics - scheduleRecurringMetricsGeneration()', function (): void {
+  afterEach('', function () {
+    metrics.stopMetrics()
+  })
+
+  it('publishes all gauges', async function () {
```

Comment:
Can we do the GIVEN/WHEN/THEN comments in these tests?

Reply:
fixed